appveyor.yml

```diff
@@ -1,21 +1,24 @@
 cache:
   - node_modules
+  - "%LOCALAPPDATA%\\Yarn"
+
+init:
+  - git config --global core.autocrlf true
```

Comment:
Well, my comment is on the exact line to which it applies.

And converting the code to CRLF on checkout always *created* bug in my experience (I'm not using Windows anymore for development, but I did it during years), because it means you are not running the source code of the project but a different one. The meaningful value IMO is `input`, not `true` (but this is irrelevant on CI as it will never commit anything). The only valid use case for this setting AFAIK is being able to open the code in Notepad (which is the only text editor I know of which does not support Unix line endings, but I would never use it to write code anymore).
Thus, it is even worse for CI: people installing Encore on Windows will *not* get the line endings converted (as npm/yarn will not use git for that, but download a tarball, and so won't be affected by your git config), meaning that converting line endings on CI would mean that the CI does not test the code that users will use.